skills_ml/algorithms/skill_extractors/freetext.py

```diff
@@ -84,3 +132,19 @@ def __init__(self, skills):
 
     def _skills_lookup(self):
         return set(self.skills)
+
+
+class FakeOccupationScopedSkillExtractor(OccupationScopedSkillExtractor):
+    """A skill extractor that takes a list of skills
+    instead of reading from a filename
+    """
+    def __init__(self, skills):
+        """
+        Args:
+            skills (dict) soc codes as keys, lists of skill names as values
+        """
+        self.skills = skills
+        super(FakeOccupationScopedSkillExtractor, self).__init__('')
```

Comment:
Oh! I wasn't aware there was a new syntax. I looked it up, and yes it looks a lot better. I'll definitely use it.